survey.html

```diff
@@ -159,4 +159,16 @@
     });
   </script>
 </body>
+
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
Do we need to also publish a special http page for google? Or will this work standalone? :)